### PR TITLE
chore: Week 11 (or so) taxonomy updates

### PR DESCRIPTION
**Pull request by aleene**
### What
Continued work on taxonomies #8253

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -5572,23 +5571,26 @@ nl:Koolzuurhoudende appelsappen
 
 <en:Apple juices
 en:Enriched apple juices
+fr:Jus de pommes enrichés
```
Comment (alexgarel) [inline taxonomies/categories.txt:5574]:
```suggestion
fr:Jus de pommes enrichis
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -5572,23 +5571,26 @@ nl:Koolzuurhoudende appelsappen
...
 nl:Verrijkte appelsappen
 #description:en:Apple juices with added vitamins and minerals.
 
 <en:Apple juices
 en:Flavoured apple juices
+fr:Jus de pomme aromatisé
```
Comment (alexgarel) [inline taxonomies/categories.txt:5580]:
```suggestion
fr:Jus de pommes aromatisés
```

keeping consistent in using plurals

**Comment by aleene** [comment]
@stephanegigandet the test seems to fail in some producers test. Can not find it and rather not touch it.

**Review by stephanegigandet — APPROVED** [review]

File: taxonomies/labels.txt [hunk 3]
```diff
@@ -4103,8 +4103,9 @@ bg:Био, биологично земеделие, биологично
...
+de:Bio, Öko, biologisch, biologische, biologischer, biologisches, Ökologisch, Kontrolliert biologisch, Kontrolliert ökologisch, biologische Landwirtschaft, ökologische Landwirtschaft, biologischer Landbau, ökologischer Landbau, aus kontrolliert biologischem Anbau, biologisch gewonnen, aus kontrolliert biologischer landwirtschaft, aus kontrolliert ökologischer erzeugung, aus kontrolliert ökologischer landwirtschaft, aus kontrolliert ökologischem anbau, aus kontrolliert ökologischem landbau, aus kontrollierter biologischer landwirtschaft, aus kontrollierter ökologischer landwirtschaft, kontrolliert biologischer anbau
+es:Ecológico, ecológica, Producto ecológico, Biológico, Producto biológico, Orgánico, Producto orgánico, Eco, Bio, Ingredientes ecológicos, Ingrediente ecológico, ingredientes procedentes de la agricultura ecológica, ingrediente procedente de la agricultura ecológica, agricultura ecológica, cumple con el reglamento de agricultura ecológica CE 2092/91, procedentes de la agricultura ecológica, procedente de la agricultura ecológica,de agricultura ecológica,de agricultura ecologica, Procedente e agricultura ecológica, de cultivo ecológico
+
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:4108]: There's an extra line feed here that separates the organic label into 2 entries.

**Pull request merged**